### ISSUE TITLE
Project is in Maintenance Mode

### ISSUE THREAD

**Issue by Swatinem** [Swatinem/rollup-plugin-dts#277]
I have decided to put this projects into *maintenance mode*, which means I will not do any active development on it anymore.

That being said, the project is stable, works well and requires little maintenance. There will still be occasional releases when necessary to support new TypeScript versions.

I also very much appreciate contributions and PRs, but might be slow to react to review and merge those.

Thanks you very much for your continued interest in this project.

**Comment by supuwoerc** [comment]
Hello, thank you very much for making this plugin, before the stable version, I found an error, before that, I used 5.3.0 version and generated files perfectly, but recently I installed 5.3.1 version, there was an error, I reduced the version to 5.3.0, error message, I found onLog in the dts product. If this is a mistake we hope we can fix it before we stabilize it. Thank you very much

<img width="949" alt="截屏2023-08-03 22 41 45" src="https://github.com/Swatinem/rollup-plugin-dts/assets/36561218/077ee4f5-941a-4212-bfa0-372d228f2205">

**Comment by supuwoerc** [comment]
> Hello, thank you very much for making this plug-in. Before the stable version, I found an error. Before that, I used version 5.3.0 and generated files perfectly, but recently I installed version 5.3.1, and there was an error. If this is a mistake we hope we can fix it before we stabilize it. Thank you very much

It's a coincidence that the new project comes exactly one hour after you released 5.3.1, and I was puzzled for a long time🥲

**Comment by Swatinem** [comment]
That seems to be a regression from https://github.com/Swatinem/rollup-plugin-dts/pull/270. @xiaoxiangmoe mind taking a look?

But I see this is only a warning, so it does not break your build?

**Comment by supuwoerc** [comment]
> That seems to be a regression from #270. @xiaoxiangmoe mind taking a look?
> 
> But I see this is only a warning, so it does not break your build?

Yes, this is a warning that doesn't affect the result of the build, but exporting such a warning is sad for perfectionists🤔

**Comment by xiaoxiangmoe** [comment]
ok，let me have a look

**Comment by xiaoxiangmoe** [comment]
@supuwoerc This is a warning for old version of rollup. Would you mind upgrade your rollup? If you mind it, maybe I should create another PR to remove this warning for old version of rollup.

**Comment by supuwoerc** [comment]
> @supuwoerc This is a warning for old version of rollup. Would you mind upgrade your rollup? If you mind it, maybe I should create another PR to remove this warning for old version of rollup.

You're the boss，Wait for your news😁

**Comment by askoufis** [comment]
While it is understandably your right to put this project into maintenance mode and not pursue active development anymore, I believe that [closing all open issues and PRs](https://github.com/Swatinem/rollup-plugin-dts/issues/246#issuecomment-1658616409) was a mistake.

Although development (from you) may stop, that won't stop consumers from consuming this plugin and then discovering a bug. Those bugs might have had an open issue in the past, but now they have been closed, reducing the visibility of these bugs for future consumers and potential contributors. This is what happened to me ([issue](https://github.com/Swatinem/rollup-plugin-dts/issues/246), [related PR that was also closed](https://github.com/Swatinem/rollup-plugin-dts/pull/261)), and may likely happen to others in the future.

**Comment by jclaveau** [comment]
As replacement there is https://github.com/qmhc/vite-plugin-dts which is maintained
